ocs_ci/utility/lvmo_utils.py

```diff
@@ -33,6 +34,8 @@ def lvmo_health_check_base():
     lvmcluster_status = oc_obj.get(constants.LVMCLUSTER)
     try:
         lvmcluster_status.get("status")["ready"]
+    except KeyError:
+        time.sleep(10)
```

Comment:
i dont think so, it is some kind of race condition that only happened once. 5 sec would also work, but i want to be safe. also, Im using sleep because other "wait_for" functions required "status" column that does not exist for lvmcluster